api/indexer.oas2.json

```diff
@@ -829,6 +849,16 @@
           "description": "\\[appidx\\] application index.",
           "type": "integer"
         },
+        "created-at-round": {
+          "description": "Round when this application was created.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "deleted-at-round": {
+          "description": "Round when this application was deleted.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
```

Comment:
Application:
* CreatedAtRound - We call it create in goal, the low level code just figures it out when there are app parameters.
* DeletedAtRound - The OnComplete enum is DELETE.

See goal app above